src/main/scss/components/_healthScore.scss

```diff
@@ -0,0 +1,31 @@
+.jenkins-healthScore--badge {
```

Comment:
That could be implemented as
```
.jenkins-healthScore {
  &--badge {
    height: 40px;
    width: 40px;

    border-radius: 50%;
    border-width: 3px;
    border-style: solid;
    border-color: var(--primary);

    display: table-cell;
    vertical-align: middle;
    text-align: center;

    background-color: var(--background);

    &:hover {
      background-color: var(--light-grey);
    }
  }

  &--top {
    border-color: var(--success);
  }

  &--middle {
    border-color: var(--warning);
  }

  &--bottom {
    border-color: var(--danger);
  }
}
```
which reduces some redundancy

Reply:
Sure.